### ISSUE TITLE
Problems with suppressing stderr with find

### ISSUE THREAD

**Issue by CptFox** [railsware/upterm#1096]
Hello,

When running a `find / -name x 2>/dev/null` from upterm, I get the error `find: 2: unknown primary or operator`, which I don't get in either Terminal or Hyper. However, if such a command is inside a bash file, it seems to run normally.

I'm running the Mac OS version. Can anybody reproduce on Linux ? Any idea on how to fix this ? #

**Comment by vlad-shatskyi** [comment]
@CptFox, it wasn't released yet, but it's fixed in master. You can try to build it, it's real simple: `npm run pack`.

**Comment by CptFox** [comment]
I really like the IDE-like autocomplete. Thanks for pointing me to building master, it seems to be quite far ahead.

BTW I feel the history is kind of a hindrance with commands like cd : it keeps suggesting relative paths I've used before in other directories, rather than suggesting directories that are actually reachable. Maybe adding some checks that the proposed paths do lead to something would help ? I'm gonna try and find if I can do that.

**Comment by vlad-shatskyi** [comment]
@CptFox thanks! I've been thinking about that too. We plan to extend history so that each history item has the directory the command was ran in. That'd allow us to filter out irrelevant directories without much overhead. Here's a relevant issue: https://github.com/railsware/upterm/issues/1128

Is there anything else that annoys you?

**Comment by CptFox** [comment]
I'd like an easy way to change color theme (I'm a sucker for Solarized Light in terminals, even though I'm usually on the Dark Side of the Theme), but that can wait.

I was gonna ask about ligature support, but it already does.

I've made a quick and dirty fix for the completion : 
- Removed the height limit on the suggestion box and get the suggestions count up to 15
- Filter history with a "if it's a cd or ls and any argument doesn't start with - or/, remove it". It's dirty, but does the work
- Slice the environment suggestion like the history ones, and put the 3 first env before the 3 first history

It's a 10 lines edit, so I've done it locally, but the detailed history seems like a better solution to me.

I had stopped using Upterm when I found my finds wouldn't work, but I guess I'm switching back 😄
